examples/keyvalue/gen-go/keyvalue/admin.go

```diff
@@ -1,4 +1,4 @@
-// Autogenerated by Thrift Compiler (1.0.0-dev)
+// Autogenerated by Thrift Compiler (0.9.3)
```

Comment:
Can you symlink `go-build`'s thrift binary in to your `PATH` so this is still the `1.0.0-dev` version which is what we've been using internally and will not have this change.